Submission of Lab 01 by BT23F05F003

akshay-ambulge opened the pull request:
I have completed lab 1 by a program of factorial using iterative and recursive ways.

akshay-ambulge committed:
Completed factorial.py of lab 1

s-m-quadri commented:
@bot-s-m-quadri status

bot-s-m-quadri commented:
## 📌 PRN: BT23F05F003

We've successfully received your submission for **Lab 01**.

**Your PRN**: `BT23F05F003`  
**Lab Number**: 01  
**Base Branch**: `lab-daa-01`

### All your submissions so far:
- #75
- #57
- #30

Please ensure you complete all required files for this lab.  
We will review your submissions **within at most 3 weeks**, i.e., by **05 September 2025**.

If you have doubts or questions, comment here, and we’ll get back to you  
once you’ve completed your work.

_Thank you for your submission._

akshay-ambulge committed:
Completed lab 1

I've completed lab 1 by solving the 3 tasks which were based on recursive and iterative problems.

akshay-ambulge commented:
@bot-s-m-quadri status

s-m-quadri commented:
**👍🏻 Okay. Looks good to me.**

Keep it up. Glad to see your attempt to trigger the bot workflow. 🙂 

@bot-s-m-quadri merge it!

bot-s-m-quadri commented:
👍 Got it, I’ll start merging and organizing your submission.
